api.planx.uk/inviteToPay/sendPaymentEmail.test.ts

```diff
@@ -133,6 +141,8 @@ describe("Send email endpoint for invite to pay templates", () => {
 
   describe("'Payment Expiry' templates", () => {
     const templates = ["payment-expiry", "payment-expiry-agent"];
-    it.todo("soft deletes the payment_request when a payment expiry email is sent");
+    it.todo(
+      "soft deletes the payment_request when a payment expiry email is sent"
```

Comment:
We have `payment_request.paid_at` which serves as a proxy for this - would that work?

Reply:
This was a linting fix and I wasn't planning on adding this test as part of this PR. Does this fit better into #1709 ?

Reply:
Apologies - I missed that this wasn't a new additional. Happy to ignore and pick it up in another PR 👍